### ISSUE TITLE
Accessible name heading creates duplicate ID

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#1192]
There are two "accessibility support" headings in the following page, with the same ID. This is caused by the "accessibility support" heading in the "accessible name" definition:

https://act-rules.github.io/rules/59796f#accessibility-support

This is a similar problem to what we saw in #1163.

**Comment by dd8** [comment]
Looks like this is happening on every rules page. Other problems occurring on every rules pages:

- using `<span class="heading">Input Aspects</span>` instead of an h1-h6 for headings (fails 1.3.1 - see F2 Example 1)
- nesting `<main>` inside `<section>` which is not nested according to spec so fails 4.1.1 (see https://html.spec.whatwg.org/multipage/grouping-content.html#hierarchically-correct-main-element)
- links using `target=_blank` opening in new windows without warning (https://www.w3.org/WAI/WCAG21/Techniques/failures/F22.html)

**Comment by Jym77** [comment]
The original issue pointed by @WilcoFiers seems to be fixed now (by #1201).
Issue pointed by @dd8 are still present.

**Comment by Jym77** [comment]
> Issues pointed by @dd8 are still present.

But these might actually be websites issues rather than rules issues. @jkodu ?

**Comment by jkodu** [comment]
There is an issue for this in the website repository - https://github.com/act-rules/act-rules-web/issues/144

I have linked this one as a reference. Closing this issue.
